pumpio/pumpio.php

```diff
@@ -870,13 +867,7 @@ function pumpio_dounlike(&$a, $uid, $self, $post, $own_id) {
 			$contactid = $orig_post['contact-id'];
 	}
 
-	$r = q("UPDATE `item` SET `deleted` = 1, `unseen` = 1, `changed` = '%s' WHERE `verb` = '%s' AND `uid` = %d AND `contact-id` = %d AND `thr-parent` = '%s'",
-		dbesc(DateTimeFormat::utcNow()),
-		dbesc(ACTIVITY_LIKE),
-		intval($uid),
-		intval($contactid),
-		dbesc($orig_post['uri'])
-	);
+	Item::delete(["`verb` = ? AND `uid` = ? AND `contact-id` = ? AND `thr-parent` = ?", ACTIVITY_LIKE, $uid, $contactid, $orig_post['uri']]);
```

Comment:
Could you please use the simple condition array form like this:
````php
['verb' => ACTIVITY_LIKE, 'uid' => $uid, 'contact-id' => $contactid, 'thr-parent' => $orig_post['uri']]
````

Reply:
Done. Was some kind of "mass replace", so I hadn't got a deeper look.